staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -31,12 +30,6 @@ const logFlushFreqFlagName = "log-flush-frequency"
 
 var logFlushFreq = pflag.Duration(logFlushFreqFlagName, 5*time.Second, "Maximum number of seconds between log flushes")
 
-// TODO(thockin): This is temporary until we agree on log dirs and put those into each cmd.
-func init() {
-	klog.InitFlags(flag.CommandLine)
-	flag.Set("logtostderr", "true")
```

Comment:
The above has been removed now that we're using `klog.InitFlags` elsewhere and `logtostderr` is already defaulting to true.

Reply:
is the overall init() block still required? that is still registering klog flags into the global command line, right?